app/src/main/kotlin/com/wire/android/ui/home/HomeScreen.kt

```diff
@@ -101,21 +114,42 @@ fun HomeContent(
         ) {
             // TODO(): Enable top search bar
             if (false) {
-                AppTopBarWithSearchBar(
-                    scrollPositionProvider = scrollPositionProvider,
-                    searchBarHint = stringResource(R.string.search_bar_hint, stringResource(id = title).lowercase()),
-                    // TODO: implement the search for home once we work on it, for now we do not care
-                    searchQuery = "",
-                    onSearchQueryChanged = {},
-                    onSearchClicked = { },
-                    onCloseSearchClicked = { },
-                    appTopBar = homeTopBar,
-                    content = {
-                        homeNavigationGraph()
-                    }
+                Scaffold(
```

Comment:
Wouldn't this work as well if we declared the Scaffold only once?

```
Scaffold(
                snackbarHost = {
                    SwipeDismissSnackbarHost(
                        hostState = snackbarHostState,
                        modifier = Modifier.fillMaxWidth()
                    )
                },
            )
            {
                Box(modifier = Modifier.padding(it)) {
                    // TODO(): Enable top search bar
                    if (false) {
                        AppTopBarWithSearchBar(
                            scrollPositionProvider = scrollPositionProvider,
                            searchBarHint = stringResource(R.string.search_bar_hint, stringResource(id = title).lowercase()),
                            // TODO: implement the search for home once we work on it, for now we do not care
                            searchQuery = "",
                            onSearchQueryChanged = {},
                            onSearchClicked = { },
                            onCloseSearchClicked = { },
                            appTopBar = homeTopBar,
                            content = {
                                homeNavigationGraph()
                            }

                        )
                    } else {
                        homeNavigationGraph()
                    }
                }
            }

Reply:
In the second Scaffold we have topBar to declare co in this case there will be another if statement in topBar. So what is better in your opinion?

Reply:
Ok. Let's leave it as it is for now. It will need to be refactored once the search bar is in place anyways.